requirements.in

```diff
@@ -1 +1,2 @@
 # Add your integration-specific dependencies here
+pydantic
```

Comment:
no need to add pydantic here since it is already included in [requirements-base](https://github.com/PADAS/gundi-integration-onyesha/blob/779e6d648ab1a682e87fcf7199c92aa7bceab2e2/requirements-base.in#L4)